client/src/components/MainPage/TranslationsPage/DonateModal/AudioTranslation/index.jsx

```diff
@@ -36,6 +72,19 @@ class AudioTranslation extends Component {
         <h3 className="file__name" ref="fileName">
           No file Choosen
         </h3>
+        <hr />
+        <h4>Or record audio</h4>
+        <FontAwesomeIcon
+          icon={faMicrophone}
+          className={`record__button ${recording ? 'recording' : null}`}
```

Comment:
Should be renamed to ```record-button``` and ```stop-button```, as they don't have direct parent with ```stop``` or ```record``` in their names.
```suggestion
          className={`record-button ${recording ? 'recording' : null}`}
```